docs/dependencies.md

```diff
@@ -0,0 +1,59 @@
+### Installing required dependencies for local build
+
+To get requirements installed locally, run:
+
+```bash
+$> make deps
+```
+
+or you need:
+
+- [`luet`](https://github.com/mudler/luet)
+- [`luet-makeiso`](https://github.com/mudler/luet-makeiso)
+- [`squashfs-tools`](https://github.com/plougher/squashfs-tools)
+  - `zypper in squashfs` on SLES or openSUSE
+- [`xorriso`](https://dev.lovelyhq.com/libburnia/web/wiki/Xorriso)
+  - `zypper in xorriso` on SLES or openSUSE
+- `yq` ([version `3.x`](https://github.com/mikefarah/yq/releases/tag/3.4.1)), installed via [packages/toolchain/yq](https://github.com/rancher-sandbox/cOS-toolkit/tree/master/packages/toolchain/yq) (optional)
+- [`jq`](https://stedolan.github.io/jq), installed via [packages/utils/jq](https://github.com/rancher-sandbox/cOS-toolkit/tree/master/packages/utils/jq) (optional)
+
+_Note_: Running `make` deps will install only `luet`, `luet-makeiso`, `yq` and `jq`. `squashfs-tools` and `xorriso` needs to be provided by the OS.
+
+### Manually install dependencies
+
+To install luet locally, you can also run as root:
+```bash
+$> curl https://get.mocaccino.org/luet/get_luet_root.sh | sh
```

Comment:
It should be advertized now `https://raw.githubusercontent.com/rancher-sandbox/cOS-toolkit/master/scripts/get_luet.sh` which installs releases-opensuse by default. It contains all the tools needed.

Reply:
fixed